sycl/include/CL/sycl/buffer.hpp

```diff
@@ -80,7 +80,7 @@ class buffer {
          const property_list &propList = {})
       : Range(bufferRange) {
     impl = std::make_shared<detail::buffer_impl>(
-        hostData, get_count() * sizeof(T), detail::getNextPowerOfTwo(sizeof(T)),
+        hostData, size() * sizeof(T), detail::getNextPowerOfTwo(sizeof(T)),
```

Comment:
Do we need this calculation? There is byte_size() method which provides the same data.

Reply:
I see that byte_size() in buffer is part of SYCL 2020 but it doesn't seem that it is implemented yet. Could you please point it out to me if this isn't what you mean?